checkout.html

```diff
@@ -0,0 +1,174 @@
+            	<option value="NE">Nebraska</option>
+            	<option value="NV">Nevada</option>
+            	<option value="NH">New Hampshire</option>
+            	<option value="NJ">New Jersey</option>
+            	<option value="NM">New Mexico</option>
+            	<option value="NY">New York</option>
+            	<option value="NC">North Carolina</option>
+            	<option value="ND">North Dakota</option>
+            	<option value="OH">Ohio</option>
+            	<option value="OK">Oklahoma</option>
+            	<option value="OR">Oregon</option>
+            	<option value="PA">Pennsylvania</option>
+            	<option value="RI">Rhode Island</option>
+            	<option value="SC">South Carolina</option>
+            	<option value="SD">South Dakota</option>
+            	<option value="TN">Tennessee</option>
+            	<option value="TX">Texas</option>
+            	<option value="UT">Utah</option>
+            	<option value="VT">Vermont</option>
+            	<option value="VA">Virginia</option>
+            	<option value="WA">Washington</option>
+            	<option value="WV">West Virginia</option>
+            	<option value="WI">Wisconsin</option>
+            	<option value="WY">Wyoming</option>
+            </select>
+          </label>
+
+          <label class="contact-csz">Zip Code
+            <input type="text" pattern="[0-9]{5}" name="customerZip" value="">
+          </label>
+        </fieldset>
+
+        <fieldset>
+          <legend>First Time Adopter?</legend>
+            <label>Yes
+              <input type="radio" name="firstTime" value="yes">
+            </label>
+            <label>No
+              <input type="radio" name="firstTime" value="no">
+            </label>
+        </fieldset>
+
+        <fieldset>
+          <legend>Pickup Location</legend>
+            <select class="" name="pickupLocation">
+              <option value="">Select</option>
+              <option value="Ballard">Ballard</option>
+              <option value="CapHill">CapHill</option>
+              <option value="Downtown">Downtown</option>
+              <option value="GreenLake">Green Lake</option>
+            </select>
+        </fieldset>
+
+        <section>
+          <h3>Dogs Adopted</h3>
+
+          <div class="cart-dog">
+            <h5>Lucas</h5>
+              <p>Cost to Adopt: $123.45</p>
+            <img src="images/lucas-checkout.jpg" alt="a picture of a black outside looking up" width="75">
+          </div>
+
+          <div class="cart-dog">
+            <h5>Duffy</h5>
+              <p>Cost to Adopt: $123.45</p>
+            <img src="images/duffy-checkout.jpg" alt="a picture of a dog outside looking up" width="75">
+          </div>
+
+        </section>
+
+        <section>
+          <h3>Total Adoption Cost</h3>
+            <p>$1234.56</p>
+        </section>
+
+        <input type="submit" name="submit" value="Submit">
+
+      </form>
+
```

Comment:
You can eliminate most, if not all of the white space in your files. Maybe a line here or there, but it's nice to compact the file a bit.